chapters/Chapter 1 — Untitled.md

```diff
@@ -18,18 +18,21 @@ Leanne turned just in time to see a crate of batteries break free of its binding
+She reached for the _Engine Power_ switch. If she turned the engines on and managed to wrestle the controls, she could right the ship and get away. But Leanne could never get away: they were always after her, and they always would be. There were at least two after her today, and there’d be at least two more tomorrow, and two more after that. 
 
-Leanne withdrew her hands. Three years ago, she had woken up in a Faces facility, bloody scalpel in hand, dead bodies scattered around her, disoriented and without memory. Naturally, she had stolen just about everything she could carry—which was quite a bit, particularly after she had stolen a cart.
+“No reprieve,” she muttered. It was necessary. Getting away was not sufficient. Thirty seven of them had to corner her on Delphinia before she had admitted it to herself: they could survive, or she could. They did not survive.
+
+Leanne withdrew her hands from the _Engine Power_ switch. Instead, she reached up under the console for the heavy, boxy backpack of a machine that was strapped underneath.
+
+Three years ago, she had woken up in a Faces facility, bloody scalpel in hand, dead bodies scattered around her, disoriented and without memory. Naturally, she had stolen just about everything she could carry—which was quite a bit, particularly after she had stolen a cart.
 
 The Faces facility had happened to be above a minor military storage office. There wasn’t much in the way of weapons, but there was a portable gravity field generator—a very unique portable gravity field generator that did not require an Alt to operate.
 
-Leanne released the belt keeping her generator in its place
-beneath the console. It leapt away, but her hand snapped out and grasped a strap. She hauled it up and flipped the switch.
+Leanne released the belt keeping her generator in its place beneath the console. It leapt away, but her hand snapped out and grasped a strap. She hauled it up and flipped the switch.
 
 The generator was very confused. It was meant to be worn, not held, and when she turned it on, it wasn’t sure which way was meant to be up. Instead of being pulled in all directions as her ship spun, Leanne was now being pulled towards the back corner of the cockpit, and now only with her normal weight. Conveniently, the back corner was exactly where she wanted to go.
 
-She let go of the rail and fell, landing on the rear cockpit wall. She lifted her suit off the wall and scrambled to put it on. She glanced at the gauges on the console: the ship’s spin was beginning to slow.
+She let go of the rail and fell, landing on the rear cockpit wall. She lifted her suit off the wall—the heavy one, just in case—and scrambled to put it on. She glanced at the gauges on the console: the ship’s spin was beginning to slow.
```

Comment:
Mention of a heavy suit here ties into later when the other assassin, in his light suit, stands no chance.